### PR TITLE
changes to be able to distinguish a tap selection from a scrolling selection…

### PR DESCRIPTION
**Pull request by frranck**
https://github.com/DroidsOnRoids/SwiftCarousel/issues/23

### CODE REVIEW

**Comment by sunshinejr** [comment]
Perfect! I would go the same way with implementing this, good job! Also, I would love if you could move conforming to `UIGestureRecognizerDelegate` into another extension, and preferably to another file, [like this one](https://github.com/DroidsOnRoids/SwiftCarousel/blob/master/Source/SwiftCarousel%2BUIScrollViewDelegate.swift).

File: Source/SwiftCarouselDelegate.swift [hunk 1]
```diff
@@ -30,10 +30,11 @@
      
      - parameter item:  Item that is selected. You can style it as you want.
      - parameter index: Index of selected item that you can use with your data source.
+     - parameter tapped: Indicate that the item has been tapped
```
Comment (sunshinejr) [inline Source/SwiftCarouselDelegate.swift]: I would add to this that if it returns true it means that it was tapped before the selection, and if it is false it was scrolled.

**Comment by sunshinejr** [comment]
Hey @frranck, is it good to go or are you gonna still change something? 😁

**Comment by frranck** [comment]
@sunshinejr I'm good

**Comment by sunshinejr** [comment]
Perfect, thank you so much! 🎉

**Comment by sunshinejr** [comment]
I've also added you as a collaborator for all the help from you 🎉

**Comment by frranck** [comment]
@sunshinejr cool! :bowtie:

**Pull request merged**